_commands/basics/bg.md

```diff
@@ -3,22 +3,35 @@
 
 bg
 ---
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
 
-<!-- minimal example -->
+`bg` resumes a stopped job in the background, so the terminal remains usable.
+
 ~~~ bash
-*short* example of command usage and output
+$ gedit file.txt
+^Z
+$ bg 'gedit file.txt'
+$ jobs
+[1]+ Running    gedit file.txt
+$
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+`bg` can be passed job numbers as well as job names
 
-##### Break it down
+~~~ bash
+$ jobs
```

Comment:
Could also be useful to see that `jobs` produces no output when it's run the first time.